Source/BugsnagSessionTracker.m

```diff
@@ -116,11 +117,21 @@ - (void)startNewSessionWithAutoCaptureValue:(BOOL)isAutoCaptured {
         return;
     }
 
-    self.currentSession = [[BugsnagSession alloc] initWithId:[[NSUUID UUID] UUIDString]
-                                                   startDate:[NSDate date]
-                                                        user:self.config.user
-                                                autoCaptured:isAutoCaptured];
+    BugsnagSession *newSession = [[BugsnagSession alloc] initWithId:[[NSUUID UUID] UUIDString]
+                                                          startDate:[NSDate date]
+                                                               user:self.config.user
+                                                       autoCaptured:isAutoCaptured];
```

Comment:
This has become separated from its first use.

Reply:
Move closer to first assignment.